webplugin/js/app/kommunicate.js

```diff
@@ -931,4 +931,27 @@ $applozic.extend(true, Kommunicate, {
             }
         }
     },
+    getCurrentPosition:() => 
+        new Promise((resolve, reject) => 
+          navigator.geolocation.getCurrentPosition(resolve, reject)
+        ),
+    getUserLocation: async function (){
+      try {
+        
+          const api_key = $applozic.fn.applozic.defaults.mapStaticAPIkey;
```

Comment:
@anujpanwar01 pls check if the same static api key can be used here, or its needs to be different?

Reply:
we can pass the googleApiKey from the script setting instead of using the default options

```
$applozic.fn.applozic.defaults.mapStaticAPIkey;

```